onadata/libs/tests/utils/test_csv_builder.py

```diff
@@ -629,26 +630,11 @@ def test_csv_export_with_labels(self):
             '_review_status', '_review_comment', '_review_date'
         ]
         self.assertEqual(expected_header, header)
-        labels = next(csv_reader)
-        self.assertEqual(len(labels), 17 + len(csv_df_builder.extra_columns))
-        expected_labels = [
```

Comment:
Why do we no longer test for `expected_labels` ?

Reply:
The list defined here is similar to that of the one defined [here](https://github.com/onaio/onadata/blob/a39361f645fa06cb4d6942b623e3b1a17a56198f/onadata/libs/tests/utils/test_csv_builder.py#L622-L631).
I assumed it was a duplication. I didnt notice they were headers vs names.
Returned that segment to the test.